doc/ethereum.md

```diff
@@ -0,0 +1,12 @@
+# Ethereum
+
+## Connecting to an Ethereum network
+
+When connecting to an Ethereum network, an ETH RPC provider needs to be specified via the `-ethUrl` flag.
```

Comment:
nit: could we replace "ETH" with "Ethereum" since ETH is more commonly used for the cryptocurrency ?

Reply:
Fixed in ef463ef